<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>钻孔电视校正工具</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="heightTab">
      <attribute name="title">
       <string>高度校正</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="currentLabel">
           <property name="text">
            <string>当前高度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="actualLabel">
           <property name="text">
            <string>实际高度</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="startButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="rectifyButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>校正</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="currentEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="connectButton">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="balanceTab">
      <attribute name="title">
       <string>白平衡</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="autoButton">
         <property name="text">
          <string> 打开</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="manualButton">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="compassTab">
      <attribute name="title">
       <string>罗盘校正</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="shiftLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>罗盘偏移：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="shiftEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getShiftButton">
         <property name="text">
          <string>获取</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shiftButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>校正</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="versionTab">
      <attribute name="title">
       <string>版本</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="versionLabel">
         <property name="text">
          <string>版本号：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="versionEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="sha1Edit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="sha1Label">
         <property name="text">
          <string>SHA1：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="getButton">
         <property name="text">
          <string>获取</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
